src/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as Manager } from './Manager';
+export { default as Target } from './Target';
+export { default as Popper } from './Popper';
+export { default as Arrow } from './Arrow';
```

Comment:
I hope this will not cause the same problem that #93 solved

Reply:
Looking at the compiled code everything looks good, anyway the old syntax is not valid at all with Rollup so there's not much choice.

Reply:
#93 was fixing a bug with the babel output not the webpack build.

You could compare the file `lib/react-popper.js` to old ones to be sure:
[react-popper.js@v0.8.0](https://unpkg.com/react-popper@0.8.0/lib/react-popper.js) <-- **Seems good** for everyone
[react-popper.js@v0.8.1](https://unpkg.com/react-popper@0.8.1/lib/react-popper.js) <-- **Wrong** ESM
[react-popper.js@v0.8.2](https://unpkg.com/react-popper@0.8.2/lib/react-popper.js) <-- **Wrong** This was when I used the same syntax as you here. 
[react-popper.js@v0.8.3](https://unpkg.com/react-popper@0.8.3/lib/react-popper.js) <-- **Seems good** for everyone